tests/unit/utils/envVars.go

```diff
@@ -0,0 +1,14 @@
+package utils
+
+import (
+	coreV1 "k8s.io/api/core/v1"
```

Comment:
minor nit - can you make the alias all lower case. That seems to be the common convention.